src/cf.data.pre

```diff
@@ -7112,11 +7112,22 @@ LOC: Config.visibleHostname
 DEFAULT: none
 DEFAULT_DOC: Automatically detect the system host name
 DOC_START
-	If you want to present a special hostname in error messages, etc,
-	define this.  Otherwise, the return value of gethostname()
-	will be used. If you have multiple caches in a cluster and
-	get errors about IP-forwarding you must set them to have individual
-	names with this setting.
+	The public domain name (FQDN) of this Proxy.
+
+	This name will be used in generated URLs sent to clients for
+	objects they need to download directly from Squid, and for other
```

Comment:
AFAICT, `and for` appears to tie the second item to the wrong place in the beginning of the sentence (`for objects and for other generated responses needing to display` instead of the intended `in generated URLs and for other generated responses needing to display`. IMHO, the intent of the second item would be better relayed by something like `and to name this proxy in generated responses (e.g., error pages)`.

I do not insist on this change.

Reply:
resolved by the other changes to this sentence.

Reply:
Not resolved. Exactly the same "for foo and for bar" problem remains.